src/main/java/com/eleks/academy/whoami/service/impl/GameServiceImpl.java

```diff
@@ -102,4 +103,19 @@ public void answerQuestion(String id, String player, String answer) {
 
 	}
 
+	@Override
+	public void leaveGame(String id, String player) {
+		var game = gameRepository.findById(id).orElseThrow(
+				() -> {
```

Comment:
no need of lambda here you can do it just declaring exception instance

Reply:
Resolved